components/alert.css

```diff
@@ -0,0 +1,43 @@
+.display-alert{
+    display: flex;
+    align-items: center;
```

Comment:
You can declare these properties and use it , instead of writing it multiple times